process/block/sovereignChainBlock.go

```diff
@@ -609,11 +609,9 @@ func (scbp *sovereignChainBlockProcessor) ProcessBlock(headerHandler data.Header
 		return nil, nil, err
 	}
 
-	for _, accounts := range scbp.accountsDB {
-		if accounts.JournalLen() != 0 {
-			log.Error("sovereignChainBlockProcessor.ProcessBlock first entry", "stack", accounts.GetStackDebugFirstEntry())
-			return nil, nil, process.ErrAccountStateDirty
-		}
+	if scbp.accountsDB[state.UserAccountsState].JournalLen() != 0 {
```

Comment:
check both peer accounts and user account.

Reply:
Reverted changes to check for all accounts in accounts db